docs/source/config/inputtransforms.rst

```diff
@@ -71,6 +71,39 @@ The decorator returns a factory function which will produce instances of
 :class:`~IPython.core.inputtransformer.StatelessInputTransformer` using your
 function.
 
+Transforming a full block
+-------------------------
+
+.. warning::
+
+    Transforming a full block at once will break the automatic detection of
+    wether a block of code is complete in interface relying on this
+    functionality, as for example terminal IPython. You will need to use a
```

Comment:
'as for example' doesn't work in English. You could use 'such as', 'including', or 'for example'.

('such as, for example, foo' is valid, but it's too verbose to read nicely here, IMO)